docs/Makefile

```diff
@@ -2,7 +2,7 @@ BUILDDIR     = build
 LINKCHECKDIR = $(BUILDDIR)/linkcheck
 SOURCEDIR    = .
 SPHINXBUILD  = sphinx-build
-SPHINXOPTS   = -a -n -W
+SPHINXOPTS   = -a -n -W --keep-going
```

Comment:
This lets Sphinx collect and report all the errors/warnings it finds instead of halting after finding/reporting the first one.